qa/L0_pytorch_unittest/test.sh

```diff
@@ -24,3 +23,13 @@ pytest -v -s $TE_PATH/tests/pytorch/test_multi_tensor.py
 pytest -v -s $TE_PATH/tests/pytorch/test_fusible_ops.py
 pytest -v -s $TE_PATH/tests/pytorch/test_fusible_ops_distributed.py
 pytest -v -s $TE_PATH/tests/pytorch/test_permutation.py
+
+# Build custom ONNX extensions for ONNX export test
+pip install onnxruntime==1.19.2
```

Comment:
```suggestion
```

Reply:
We don't need this do we?

Reply:
We still need the ONNX Runtime. There's no available distribution for ORT 1.13.1, so I've upgraded to 1.19.2.